docs/Rollup_basics.md

```diff
@@ -13,42 +13,45 @@ and lower cost than on-chain Ethereum contracts.
 Despite running off-chain, ArbChains are _trustless_, meaning that any
 party can force an ArbChain to execute correctly according to the code of its contracts.
 
-You can make as many ArbChains as you want.
-Typically, you'll want one ArbChain for each dapp you want to create.
+You can make as many ArbChains as you want. Each ArbChain provides the same interface as the Ethereum blockchain, and contracts can be deployed on the chain at anytime using the `CREATE` and `CREATE2` opcode.
+> Note: Earlier versions of Arbitrum did not fully support dynamic contract launching and required code for all contracts to be provided when initially launching the chain. We have now removed this limitation.
+
+Applications that reside on the same ArbChain can make synchronous calls to one another exactly as they would on Ethereum.
+Typically, you'll want to group dApps together into a single ArbChain if they need to interact and compose with one another. Applications that do not require interaction with others can be deployed on their own ArbChain.
```

Comment:
Suggest saying that applications that do not require *synchronous* interaction will often be deployed on separate chains.